.github/workflows/main.yml

```diff
@@ -40,7 +40,7 @@ jobs:
           pipeline/${{ env.DOCKERFILE }}
         build-args: |
           WF_RESOURCES=${{ inputs.workflow_id }}/
-          QUARKUS_EXTENSIONS=org.kie.kogito:kogito-addons-quarkus-jobs-knative-eventing:999-SNAPSHOT
+          QUARKUS_EXTENSIONS=org.kie.kogito:kogito-addons-quarkus-jobs-knative-eventing:999-SNAPSHOT,org.kie.kogito:kogito-addons-persistence-jdbc:9.99.0.redhat-00007,io.quarkus:quarkus-jdbc-postgresql:3.2.9.Final,io.quarkus:quarkus-agroal:3.2.9.Final
```

Comment:
is there a need to enable the `earlyaccess` repo?

https://maven.repository.redhat.com/earlyaccess/all/org/kie/kogito/kogito-addons-quarkus-persistence-jdbc/9.99.0.redhat-00007/

Reply:
yes, I was about to ask about that

Reply:
@masayag I added an ugly hack in the dockerfiles and now it should work (locally with make it works)